### ISSUE TITLE
DAP violation FileUploaderButton missing label

### ISSUE THREAD

**Issue by nictownsend** [carbon-design-system/carbon#6027]
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships SC 1.3.1

![image](https://user-images.githubusercontent.com/47386186/81181792-ddfd6080-8fa4-11ea-98e0-c337af31edca.png)

https://codesandbox.io/s/ancient-dew-x0ywi

**Comment by nictownsend** [comment]
Looks related to #5739 - @abbeyhrt did you raise it with DAP?

**Comment by joshblack** [comment]
cc @dakahn do you have a sec to confirm this week?

**Comment by dakahn** [comment]
I am unable to recreate this DAP violation in either our latest or our master branches @nictownsend what DAP ruleset are you running and what is your Carbon version number?

**Comment by nictownsend** [comment]
Looks likely it's been fixed upstream since I raised this
